src/main/java/hudson/plugins/git/extensions/impl/CloneOption.java

```diff
@@ -49,13 +59,18 @@ public void decorateCloneCommand(GitSCM scm, Run<?, ?> build, GitClient git, Tas
             listener.getLogger().println("Using shallow clone");
             cmd.shallow();
         }
+        if (noTags) {
+            listener.getLogger().println("Avoid fetching tags");
+            cmd.tags(false);
+        }
         cmd.timeout(timeout);
         cmd.reference(build.getEnvironment(listener).expand(reference));
     }
     
     @Override
     public void decorateFetchCommand(GitSCM scm, GitClient git, TaskListener listener, FetchCommand cmd) throws IOException, InterruptedException, GitException {
-    	cmd.timeout(timeout);
+        cmd.tags(!noTags);
```

Comment:
It was three spaces, now it's four, and right next to a modified line. Is that really how strictly you enforce "no unrelated formatting changes"?